bin/eslint_d.js

```diff
@@ -34,10 +34,17 @@ if (cmd === 'start') {
       var useStdIn = (process.argv.indexOf('--stdin') > -1);
 
       if (useStdIn) {
-        var concat = require('concat-stream');
-        process.stdin.pipe(concat({ encoding: 'string' }, function (text) {
+        var chunks = [];
+        process.stdin.setEncoding('utf8');
+
+        process.stdin.on('data', function (chunk) {
+          chunks.push(chunk.toString());
```

Comment:
[The Stream.setEncoding documentation](https://nodejs.org/api/stream.html#stream_readable_setencoding_encoding) says

> Call this function to cause the stream to return strings of the specified encoding instead of Buffer objects.